src/impl_/extract_argument.rs

```diff
@@ -55,6 +66,9 @@ where
 {
     type Holder = T::Holder;
 
+    #[cfg(feature = "experimental-inspect")]
+    const INPUT_TYPE: &'static str = "typing.Any | None";
```

Comment:
Can we use the `const_concat` macro here, for something like

```rust
```suggestion
    const INPUT_TYPE: &'static str = const_concat!("typing.Union[", T::INPUT_TYPE, ", None]");
```

Reply:
Sadly it is not possible to use generic parameters inside of const expression. We would need the [`generic_const_exprs`](https://github.com/rust-lang/rust/issues/76560) (or a simplified version of it) to be stable.

Reply:
A future improvement to come, I guess 😄